src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(operand) {
+      calculator.result += operand;
+    },
+    subtract(operand) {
+      calculator.result -= operand;
+    },
+    multiply(operand) {
+      calculator.result *= operand;
+    },
+    divide(operand) {
+      calculator.result /= operand;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
```

Comment:
```suggestion
      return  this;
```
не нужно хардкодить название объекта